README.md

```diff
@@ -1,24 +1,30 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# Dungeonball
+An API for retrieving the theoretical D&D statistics of your favorite NFL players, and forming parties of NFL players!
+
+### Setting Up
+To use this API for your own personal use, please follow the proceeding steps:
+
+1. Fork this repository.
+2. Clone this repository.
+3. Run `bundle install`.
+4. Run `bundle exec rails db:{drop,create,migrate}`.
+5. Run `rails csv_load:players`.
+6. Run `rails s`.
+7. Enjoy!
+
+### Endpoints
+* ` GET /api/v1/players ` - This endpoint searches through a list of all available players, and returns a list of all players with the queryed name.
+Requires a query parameter ```name``` for a player's name.
+* ` GET /api/v1/players/:id ` - Retrieves the individual player statistics of the requested player, based on their ID within the database.
+* ` GET /api/v1/parties/:id ` - Retrieves the requested user party, assuming it has received both a valid party ID and a valid query parameter `user_id` for the user who owns the party.
+* ` POST /api/v1/parties/:id/players ` - Adds a player to a selected party, assuming the current user is the owner of the party.  Requires a `user_id` query parameter and a `player_id` query parameter.
+
```

Comment:
I'm now realizing that we don't need a user_id for this last endpoint - to add a player to a party we will only need the player_id and party_id so we can create a PartyPlayer object 😁